.editorconfig

```diff
@@ -13,7 +13,7 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
-line_length = 88
+line_length = 96
 multi_line_output = 4
```

Comment:
Please make sure that the other isort options are correct too. Can we remove these maybe at all as they are in the setup.cfg/pyproject.toml anyway? (all besides line length)